tests/driver_ili9341/riot_logo.h

```diff
@@ -0,0 +1,71 @@
+const uint16_t picture[69][128] = {
```

Comment:
Instead of `69`, I had to use `70` with an extra empty line at the end of array to avoid having an alone green pixel at the bottom right of the logo.
I have no idea why it was there. One could think of a division by 2 issue but there is no such thing in `ili9341_map` implementation.

Reply:
It's an off-by-one here: https://github.com/RIOT-OS/RIOT/pull/9948/files#diff-601d0bbb085a269ae7e4f7d2ac5fe68bR257-R258

Reply:
And fixed, I was able to reproduce your issue and it should be fixed in the latest iteration.